staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -39,6 +39,15 @@ const (
 
 const HotplugDiskDir = "/var/run/kubevirt/hotplug-disks/"
 
+type ErrorPolicy string
+
+const (
+	ErrorPolicyStop     ErrorPolicy = "stop"
+	ErrorPolicyIgnore   ErrorPolicy = "ignore"
+	ErrorPolicyReport   ErrorPolicy = "report"
+	ErrorPolicyEnospace ErrorPolicy = "enospace"
+)
+
```

Comment:
IMHO, `ErrorPolicy` is way too generic, and it is not clear from the name what it refers to. Maybe:
```suggestion
type DiskErrorPolicy string

const (
	DiskErrorPolicyStop     ErrorPolicy = "stop"
	DiskErrorPolicyIgnore   ErrorPolicy = "ignore"
	DiskErrorPolicyReport   ErrorPolicy = "report"
	DiskErrorPolicyEnospace ErrorPolicy = "enospace"
)

```
WDYT?